notebooks/src/el-paso.ipynb

```diff
@@ -558,27 +604,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "@question(4204)\n",
-    "def peak_icu_admissions():\n",
+    "@question(4204, community_weight=0.3)\n",
```

Comment:
Ideally we'd only mix in the community distribution when we don't have historical data, right? Same for next question. (Fine not to do as part of this PR)

Reply:
yeah I think that's right. Definitely planning to revisit mixing in community prediction soon, but actually maybe it would make sense to not mix it in for historical data as part of this PR

Reply:
(addressed here for these questions, added to https://github.com/oughtinc/ergo/issues/217 for other questions)